src/go/accessd/server/server.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2021 The Magma Authors.
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree.
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package server
+
+import (
+	"context"
+	"net"
+	"time"
+
+	"github.com/pkg/errors"
+	"google.golang.org/grpc"
+
+	"github.com/magma/magma/log"
+	"github.com/magma/magma/protos/magma/sctpd"
+	service_sctpd "github.com/magma/magma/service/sctpd"
+)
+
+const (
+	mmeUpstreamTarget  = "unix:///tmp/mme_sctpd_upstream.sock"
+	mmeGrpcDialTimeout = time.Second
+)
+
+func newMmeGrpcConn() (*grpc.ClientConn, error) {
+	ctx, _ := context.WithTimeout(context.Background(), mmeGrpcDialTimeout)
+	return grpc.DialContext(ctx, mmeUpstreamTarget, grpc.WithInsecure(), grpc.WithBlock())
+}
+
+func newServer(logger log.Logger) *grpc.Server {
+	grpcServer := grpc.NewServer()
+
+	mmeGrpcConn, err := newMmeGrpcConn()
+	if err != nil {
+		panic(err)
+	}
+	sctpdUplinkServer := service_sctpd.NewProxyUplinkServer(logger, mmeGrpcConn)
+	sctpd.RegisterSctpdUplinkServer(grpcServer, sctpdUplinkServer)
+
+	return grpcServer
+}
```

Comment:
there are too many comments so it is difficult for me to understand what it is here, but it looks like you are creating a GRPC server.

Why don't we use/extend the orc8r framework to create services? We don't have to worry for config files or how to identify each service. it is all included on the framework see [here](http://github.com/magma/magma/blob/cdc9dd678ccb35e9dd24c8b9a4aefc2894444dba/feg/gateway/services/s8_proxy/s8_proxy/main.go#L33-L33)

This framework also allows us to create GRPC test services. This way we don't need to inject mocks inside the code. We just build a complete mock service. For example that is what we did [here](http://github.com/magma/magma/blob/cdc9dd678ccb35e9dd24c8b9a4aefc2894444dba/feg/gateway/services/aaa/servicers/accounting_test.go#L41-L41) were the AAA server communicates directly with pipelined. In that case, instead of having to inject a grpc mocked client and polluting the code, we just create our *own mock pipelined grpc service*

Reply:
Regarding the too many comments @uri200, hit the `i` key and it will toggle all comments!

Reply:
will look at the orc8r code for creating services, but after we figure out the DI strategy (currently doing some manual DI, but exploring Fx next)